planner/core/explain.go

```diff
@@ -272,6 +352,33 @@ func (p *PhysicalHashJoin) ExplainInfo() string {
 	return buffer.String()
 }
 
+// ExplainNormalizedInfo implements Plan interface.
+func (p *PhysicalHashJoin) ExplainNormalizedInfo() string {
+	buffer := new(bytes.Buffer)
+
+	if len(p.EqualConditions) == 0 {
+		buffer.WriteString("CARTESIAN ")
+	}
+
+	buffer.WriteString(p.JoinType.String())
+	fmt.Fprintf(buffer, ", inner:%s", p.Children()[p.InnerChildIdx].ExplainID())
+	if len(p.EqualConditions) > 0 {
+		fmt.Fprintf(buffer, ", equal:%s", expression.SortedExplainNormalizedScalarFuncList(p.EqualConditions))
+	}
+	if len(p.LeftConditions) > 0 {
+		fmt.Fprintf(buffer, ", left cond:%s", expression.SortedExplainNormalizedExpressionList(p.LeftConditions))
+	}
+	if len(p.RightConditions) > 0 {
+		fmt.Fprintf(buffer, ", right cond:%s",
+			expression.SortedExplainNormalizedExpressionList(p.RightConditions))
+	}
+	if len(p.OtherConditions) > 0 {
+		fmt.Fprintf(buffer, ", other cond:%s",
+			expression.SortedExplainNormalizedExpressionList(p.OtherConditions))
+	}
+	return buffer.String()
+}
+
 // ExplainInfo implements Plan interface.
```

Comment:
Why not add a explainInfo function here? It seems more consistent with previous code.